setup.py

```diff
@@ -9,12 +9,15 @@
       url="https://github.com/brendonh/schemup",
       long_description=open('README.md').read(),
 
-      packages = find_packages(),
+      packages=find_packages(),
 
-      install_requires = [
+      install_requires=[
+            "six==1.15.0",
+            "PyYAML==5.4.1",
+            "psycopg2-binary==2.8.6"
       ],
 
-      package_data = {
+      package_data={
       },
 
-)
+      )
```

Comment:
Closing parenthesis should have the same indentation as the opening one.
Also, I'd recommend we indent & list all args (`name` here) so it's easier to read.